contracts/common/TimeHelpers.sol

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity ^0.4.18;
+
+
+contract TimeHelpers {
+    /**
+    * @dev Returns the current block number.
+    *      Using a function rather than `block.number` allows us to easily mock the block number in
+    *      tests.
+    */
+    function getBlockNumber() internal view returns (uint256) {
+        return block.number;
+    }
+
+    /**
+    * @dev Returns the current block number, converted to uint64.
+    *      Using a function rather than `block.number` allows us to easily mock the block number in
+    *      tests.
+    */
+    function getBlockNumber64() internal view returns (uint64) {
+        return uint64(block.number);
```

Comment:
On these conversions, can we use the new `Uint256Helpers`?

Reply:
Yes, we can, and I thought about it, but I considered unnecessary to import another library as I thought if block number or timestamp were ever bigger than `2^64` (`~1.8e19`, for time almost `6e11` years) this would be a minor problem. ;-)

Reply:
Despite being improbable, I think it's best to use the safe conversion. We use SafeMath when incrementing a lot of these time-based variables in the apps so we should apply the same rigour here.

Reply:
Ok, I'll change it, although I kind of disagree. I don't remember by heart all that occurrences, but I think they mostly involve "external" data (parameters, computed, etc), while here we are just converting "system" values (`block.number` and `now`) so I think it's pointless.